### PR TITLE
newsletter 2019-08-02

### PR DESCRIPTION
**Pull request by ryan-shea**

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -932,8 +932,8 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "Grow Ethereum Hackathon: Just Days Away"
-    new_kudos_pks = [153, 66, 4281]
+    subject = "Hacking, Begin! Grow Ethereum Has Started"
```
Comment (owocki) [inline app/retail/emails.py]: just `The Grow Ethereum  Hackathon Has Started` seems simpler..

File: app/retail/emails.py [hunk 2]
```diff
@@ -972,45 +972,45 @@ def render_new_bounty_roundup(to_email):
...
 
     bounties_spec = [{
-        'url': 'https://github.com/gitcoinco/web/issues/4744',
-        'primer': 'Allow Ability to Resurface Bounties to the Top When Expiration Date is Extended',
+        'url': 'https://github.com/fluencelabs/Bounties/issues/1',
+        'primer': 'Help Build The Real Decentralized Web',
     }, {
-        'url': 'https://github.com/SpeckleOS/speckle-browser-extension/issues/89',
-        'primer': 'Global Settings (Change Colour Theme and Log Out)',
+        'url': 'https://gitcoin.co/issue/PegaSysEng/BountiedWork/20/3281',
```
Comment (danlipert) [inline app/retail/emails.py]: Remember these have to be github links, not gitcoin @ryan-shea

File: app/retail/emails.py (hunk 1 above)
Comment (ryan-shea) [reply app/retail/emails.py]: fixed

File: app/retail/emails.py (hunk 2 above)
Comment (ryan-shea) [reply app/retail/emails.py]: fixed. my bad.

**Pull request closed without merging**